build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -267,38 +291,14 @@ public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
             {
                 Command.Create(Path.Combine(pkgDir, "pack.cmd"))
```

Comment:
nit: This can just call Exec now, and so we don't need multiple code paths.  Could just say:

`Exec(Path.Combine(pkgDir, "pack"))`

(or it could put the extension in based on windows or not).